issue-2: yarn build failed

```
ReferenceError: window is not defined
[ERROR] Unable to build website for locale en.
[ERROR] Error: Failed to compile with errors.
    at D:\REACT-DEV\....\node_modules\@docusaurus\core\lib\webpack\utils.js:180:24
    at D:\REACT-DEV\....\node_modules\webpack\lib\MultiCompiler.js:554:14
    at processQueueWorker (D:\REACT-DEV\....\node_modules\webpack\lib\MultiCompiler.js:491:6)
    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)
```

after follow your tutorial, i have facing this problem, when i create some file in theme folder.

any clue?

--- a/package.json
+++ b/package.json
@@ -20,10 +20,10 @@
     "clsx": "^1.2.1",
     "docusaurus-plugin-dotenv": "^1.0.1",
     "firebase": "^9.22.0",
-    "firebaseui": "^6.0.2",
     "prism-react-renderer": "^1.3.5",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "react-firebaseui": "^6.0.0"
   },
   "devDependencies": {
     "@docusaurus/module-type-aliases": "2.4.1"

--- a/src/components/Login/index.js
+++ b/src/components/Login/index.js
@@ -1,11 +1,11 @@
-import React, { useEffect } from "react";
-
-import firebase from "firebase/compat/app";
-import * as firebaseui from "firebaseui";
+import React from "react";
 
 import { auth } from "../Auth";
 
-import 'firebaseui/dist/firebaseui.css';
+import StyledFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth';
+import firebase from 'firebase/compat/app';
+import 'firebase/compat/auth';
+
 import './styles.css';
 
 // Configure FirebaseUI.
@@ -16,10 +16,7 @@ export const uiConfig = {
 };
 
 export function Login() {
-    useEffect(() => {
-        const ui = firebaseui.auth.AuthUI.getInstance() || new firebaseui.auth.AuthUI(auth);
-        ui.start('.auth-wrapper', uiConfig);
-    });
-
-    return <div className="auth-wrapper"></div>;
+    return <div className="auth-wrapper">
+        <StyledFirebaseAuth uiConfig={uiConfig} firebaseAuth={auth} />
+    </div>;
 };

